### PR TITLE
HPCC-32163 Zap LogFilter Absolute Time Range Fix

### PR DESCRIPTION
**Pull request by rpastrana**
- Fixes absolute time range param name
- Fixes erroneous time range param test

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: esp/services/ws_workunits/ws_workunitsHelpers.hpp [hunk 1]
```diff
@@ -206,9 +208,9 @@ struct WUComponentLogOptions
             logDataFormat = logAccessFormatFromName(requestedLogDataFormat.str());
 
         StringBuffer start; // Absolute query time range start in YYYY-DD-MMTHH:MM:SS
-        zapHttpRequest->getParameter("LogFilter_AbsoluteTimeRange_Start", start);
+        zapHttpRequest->getParameter("LogFilter_AbsoluteTimeRange_StartDate", start);
```
Comment (rpastrana) [inline esp/services/ws_workunits/ws_workunitsHelpers.hpp:211]: @jeclrsg I noticed the zap request absolute time range hasn't been working, and noticed this discrepancy in the param name. Can you confirm this is the correct name? This is targeting 9.2.x and it needs to be in synch w/ eclwatch for all versions.

Comment (jeclrsg) [reply esp/services/ws_workunits/ws_workunitsHelpers.hpp:211]: Yes, LogFilter_AbsoluteTimeRange_StartDate and LogFilter_AbsoluteTimeRange_EndDate are correct

Comment (rpastrana) [reply esp/services/ws_workunits/ws_workunitsHelpers.hpp:211]: Thanks @jeclrsg has that been the case since 9.2.x?

**Comment by rpastrana** [comment]
@asselitx quick ESP code review.

File: esp/services/ws_workunits/ws_workunitsHelpers.hpp (hunk 1 above)
Comment (jeclrsg) [reply esp/services/ws_workunits/ws_workunitsHelpers.hpp:211]: That's right, everything 9.2.x and up

**Review by asselitx — APPROVED** [review]

**Review by jeclrsg — APPROVED** [review]

**Pull request merged**